src/main/java/org/iot/dsa/dslink/java/v2/jdbc/Container.java

```diff
@@ -0,0 +1,42 @@
+package org.iot.dsa.dslink.java.v2.jdbc;
+
+/**
+ * Provides an easy way to set variables from an inner class to an
+ * outer class.
+ *
+ * @author Samuel Grenier
+ */
+public class Container<T> {
```

Comment:
hum, I'm not sure what's the use of this. but I don't particularly like the name.. It doesn't tell anything useful.

Reply:
It's just a container. Basically used to go around java static checking in member classes.

Not being used, but might be necessary for implementing streaming queries.